lib/DDG/Spice/Time.pm

```diff
@@ -22,21 +22,23 @@ triggers any => "time";
 
 my $capitals = Load(scalar share("capitals.yml")->slurp);
 
-my $place_connector = join '|', qw(in of for at);
-
 handle query_lc => sub {
     my $q = shift;
 
-    return unless $q =~ m/^(what'?s?|is|the|current|local|\s)*time(?:is|it|\s)*(?:\b$place_connector\b)\s+(?<loc>[^\?]+)[\?]?$/;
-    $q = $+{loc};
-    trim($q);
-    $q =~ s/,//g;
+    $q =~ m/(?<rest>what'?s?|is|the|current|local|\s)*time(?:is|it|in|of|for|at|\s)*(?<loc>[^\?]*)[\?]*$/;
```

Comment:
You need to keep the `$place_connector` because queries such as `what time is it in india` don't work anymore (please put that in the test as well).